project/src/test/java/LogoutTests.java

```diff
@@ -0,0 +1,24 @@
+import base.BaseTestAfterLogin;
+import base.BaseTestWithDriver;
+import data.ui.MenuItem;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import steps.DriverSteps;
+import steps.LoginSteps;
+import steps.NavigationSteps;
+import utils.log.LogProvider;
+
+public final class LogoutTests extends BaseTestAfterLogin implements LogProvider {
+
+	private LoginSteps loginSteps = getUiStepsComponent().loginSteps();
+	private NavigationSteps navigationSteps = getUiStepsComponent().navigationSteps();
+
+	@Test
+	public void logoutFromApp() {
+		navigationSteps.logoutUser();
+		Assert.assertTrue(loginSteps.isScreenActive(),"Login screen is NOT active after logout from application.");
```

Comment:
invalid Assert class used: should be utils.runner.Assert, but was org.testng.Assert (see import section).

Let's use single standard of assertion methods "something should be": "Login screen should be active after logout". Then, if assertion fails, it will read: Login screen should be active after logout. expected: true, but was: false.